### ISSUE TITLE
Improve the command/subcommand naming

### ISSUE THREAD

**Issue by git-hulk** [apache/kvrocks-controller#19]
Below is the commands of the Kvrocks cluster client:
```
List of common commands:
   ls              -  list namespace or cluster
   cd              -  change dir between  namespcae and cluster
   rm              -  rm current namespaces or rm current cluster

List of namespcae commands:
   mkns            -  make namespcae

List of cluster commands:
   mkcl            -  make cluster
   showcluster     -  show cluster topo info
   showfailover    -  show failover tasks
   showmig         -  show migrate tasks
   psynctopo       -  sync topo to cluster nodes
   predisdo        -  do redis command to cluster nodes

List of shard commands:
   addshard        -  add shards
   delshard        -  del shard
   migdata         -  migrate slots and data
   migslot         -  migrate slots only not include data

List of node commands:
   addnode         -  add node
   delnode         -  del node
   failover        -  failover node
   synctopo        -  sync node topo to node
   redisdo         -  do redis command to node
```

**Comment by git-hulk** [comment]
From my side, I think can do those changes:
```shell
mkns => create_namespace
mkcl => create_cluster
showcluster => show_cluster
showfailover => show_failover_tasks
showmig => show_migrate_tasks
psynctopo => sync_topo_to_cluster
predisdo => send_command_to_cluster
addshard => add_shard
delshard => del_shard
migdata => migrate_slot_and_data
migslot => migrate_slot_only
addnode => add_node
delnode => del_node
synctopo => sync_topo_to_node
redisdo => send_command_to_node
```

Even thought it looks a bit wordy at first galance, but I think
it will be better to make it clearly.

HDYT, @joeylichang @ShooterIT

**Comment by ShooterIT** [comment]
I think non-shorthand is better, it is much clear, and we already provide `help` option to let users learn these informations.

Current names are look good to me.

**Comment by ShooterIT** [comment]
I ever tested couchbase, i think its' cli is good, FYI: https://docs.couchbase.com/server/current/cli/cbcli/couchbase-cli.html

**Comment by git-hulk** [comment]
> I ever tested couchbase, i think its' cli is good, FYI: https://docs.couchbase.com/server/current/cli/cbcli/couchbase-cli.html

cool

**Comment by git-hulk** [comment]
closed by 481c411bd78947e8e0ac5c9e08eb87fff312898d
